contracts/deployChecklist.txt

```diff
@@ -0,0 +1,5 @@
+Check gas price
+Check zeppelin sdk is setup correct
+Check that the first iteration is the desired length
+Delete artifacts and build folder
+Validate testnet deployment
```

Comment:
Just thought I'd jot this down. If we come up with anything more on deploy we can add it.

Reply:
Great.